src/localconf.js

```diff
@@ -61,6 +62,7 @@ function buildConfig(opts) {
     workspaceURL,
     manifestServiceURL,
     requestorURL,
+    dicomViewerURL,
```

Comment:
remove these if they are not being used?

Reply:
🤦‍♀️